prime-angular-frontend/src/app/modules/adjudication/shared/components/enrollee-table/enrollee-table.component.html

```diff
@@ -183,6 +183,12 @@
                           matTooltip="Submission confirmed"
                           matTooltipPosition="after">verified
                 </mat-icon>
+                <mat-icon *ngIf="row.linkedEnrolleeId || row.possiblePaperEnrolmentMatch"
+                          class="paper-submission-icon"
+                          [class]="row.linkedEnrolleeId ? 'linked' : 'possible-match'"
+                          [matTooltip]="row.linkedEnrolleeId ? 'Paper Submission Linked' : 'Possible Paper Enrolment Matches'"
+                          matTooltipPosition="after">task
```

Comment:
I'm not really onboard with using the same icon with a different colour and tooltip. Something I don't think we have really done.

I would make the suggestion to instead use a different icon like find_in_page. Has a magnifying glass instead of check mark. and then keep the colouring how it is with the rest of the icons that they change with the status.

Reply:
Good advice but this would change the acceptance criteria for PRIME-1900. Could become a future improvement ticket.

Reply:
Sorry I fail to see how this solution reflects the acceptance criteria and mockups. The mockups fail to show the 
``` possibleMatch && unlinked ```
state and the only reference in the criteria is as follows
```a. Multiple results unlinked icon to yellow```

So this would lead me to believe we should have an "unlinked" icon. 

Also with our current state of the admin all icons are matched to the colour of the status. This is how currently every icon is done. 
![image](https://user-images.githubusercontent.com/59669288/143964873-31cab115-1a23-487c-8db7-69a3e6f5f523.png)
So looking at this an admin might assume that the first enrollee is linked and all good since it has the "linked" icon and in the review state. Since the colour of the icon could be somewhat hidden by the presumption of the status being the same colour. 

So if we through all other facets of admin used different icons to display different "states" and colour is inherited from the status, changing those presumptions for this one thing is wrong and an oversite of this ticket.

Reply:
@hogmann I agree, but that's what they wanted and had us write up in the ticket during a refinement.  It wasn't our teams idea to change the colour and at the time we questioned it since it is based on status, but Paper Enrollees are considered "different".  One ticket in the backlog is to change the status message on Paper Enrolments since a Paper Enrollee can never be "Under Review" it is either "Incomplete" (being filled in by Admin) or "Completed" (submitted and finalized) making it unrelated to any status.  The only suggestion of ours that made it in was the alert at the top with a message to flush out the meaning.

Reply:
Sounds good ill close this conversation, but will probably bring it up down the road again with that ticket.